### PR TITLE
Add Gmail / Spotlight Shortcuts & Citrix Ignore

### PR DESCRIPTION
**Pull request by aperlstein**
Added Gmail shortcuts CTRL+ENTER and one with SHIFT Modifier for FrontApp
Added Spotlight Search CTRL+SPACE
Added Ignore for a few shortcuts for Citrix

### CODE REVIEW

File: windows_shortcuts.json [hunk 1]
```diff
@@ -107,6 +107,93 @@
...
+                "^com\\.citrix\\.XenAppViewer$"
+              ]
+            }
+          ],
+          "type": "basic"
+        }
+      ]
+    },
+    {
+      "description": "CTRL+SPACE Spotlight Search",
```
Comment (rux616) [inline windows_shortcuts.json]: Please change the description to match the existing format.

File: windows_shortcuts.json [hunk 2]
```diff
@@ -107,6 +107,93 @@
...
+                "^com\\.citrix\\.XenAppViewer$"
+              ]
+            }
+          ],
+          "type": "basic"
+        }
+      ]
+    },
+    {
+      "description": "CTRL+ENTER Gmail",
```
Comment (rux616) [inline windows_shortcuts.json]: Please change the description to match the existing format.

File: windows_shortcuts.json [hunk 3]
```diff
@@ -107,6 +107,93 @@
         }
       ]
     },
+    {
+      "description": "CTRL+SHIFT+ENTER Gmail",
```
Comment (rux616) [inline windows_shortcuts.json]: Please change the description to match the existing format.

**Comment by rux616** [comment]
Please update the README document with your additions.

**Comment by Geczy** [comment]
merge please

**Comment by rux616** [comment]
#37 integrates this PR via jsonnet. Closing.

**Pull request closed without merging**